src/site/_includes/partials/feedback.njk

```diff
@@ -0,0 +1,80 @@
+<web-feedback>
```

Comment:
Can we put this inside Feedback instead? It doesn't feel like to me that it belongs in the generated source (although I'll admit the line is a bit fuzzy).

We're moving to SD so this will be less awkward soon, but right now maybe just set `.innerHTML` on the `<web-feedback>` on startup?

Reply:
Had a chat with Sam about this.

I think what I'd recommend is moving all of this content into the `render()` function for `web-feedback`. I think this element probably wouldn't use Shadow DOM even if we had that option because it's not doing any slotting so there isn't much benefit of obscuring the styles and having to use adoptedStyleSheets. Instead, it can call render, splat out all this content as light dom children, and take advantage of the site's already existing stylesheets.